<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1623</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Trees</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #EAE7DC</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Form_tree">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>20</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
              margin: 5px;
              padding: 8px 15px;
              border-radius: 12px;
              border: 1px solid rgba(0,0,0,0.15);
              background-color:  #47A76A;
              font-size:18px;
}
QPushButton::hover {
               background-color: #B8B8FF
}</string>
    </property>
    <property name="text">
     <string>Tree</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1490</x>
      <y>20</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Branches</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Count_nodes">
    <property name="geometry">
     <rect>
      <x>1480</x>
      <y>50</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #FFF</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1281</width>
      <height>561</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #FFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Binary_Btn">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>340</y>
      <width>211</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
              margin: 5px;
              padding: 8px 15px;
              border-radius: 12px;
              border: 1px solid rgba(0,0,0,0.15);
              background-color:  #47A76A;
              font-size:18px;
}
QPushButton::hover {
               background-color: #B8B8FF
}</string>
    </property>
    <property name="text">
     <string>Balance</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Average_Btn">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>440</y>
      <width>211</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
              margin: 5px;
              padding: 8px 15px;
              border-radius: 12px;
              border: 1px solid rgba(0,0,0,0.15);
              background-color:  #47A76A;
              font-size:18px;
}
QPushButton::hover {
               background-color: #B8B8FF
}</string>
    </property>
    <property name="text">
     <string>Middle value</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1623</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
